pages/api/news-letter.ts

```
@@ -28,7 +28,8 @@ export default async function handler(
       }
 
       const supabase = getSupabaseClient();
-      await supabase.from('news_letters').upsert({email});
+      const payload: NewsLetterInsert = {email};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)